checkov/common/runners/runner_registry.py

```diff
@@ -368,7 +368,8 @@ def get_fail_thresholds(config: argparse.Namespace, report_type: str) -> _ExitCo
                     soft_fail = enf_rule.is_global_soft_fail()
                     logging.debug(f'Using enforcement rule hard fail threshold for this report: {hard_fail_threshold.name}')
                 else:
-                    logging.debug(f'Use enforcement rules is TRUE, but did not find an enforcement rule for report type {report_type}, so falling back to CLI args')
+                    sanitized_report_type = SecretsOmitter.omit_secret(report_type)
+                    logging.debug(f'Use enforcement rules is TRUE, but did not find an enforcement rule for report type {sanitized_report_type}, so falling back to CLI args')
```

Comment:
`report_type` is not sensitive and stores the name of the framework, ex. terraform, etc.